tests/unit/conftest.py

```diff
@@ -184,6 +184,126 @@ def sample_get_response_data():
         "infoUri": "http://coordinator:8080/query.html?20210817_140827_00000_arvdv",  # NOQA: E501
     }
 
+@pytest.fixture(scope="session")
+def sample_get_response_data_none():
+    """
+    This is the response to the second HTTP request (a GET) from an actual
+    Trino session. It is deliberately not truncated to document such response
+    and allow to use it for other tests. After doing the steps above, do:
+
+    ::
+        >>> cur.fetchall()
+
+    """
+    yield {
+        "id": "20210817_140827_00000_arvdv",
+        "nextUri": "coordinator:8080/v1/statement/20210817_140827_00000_arvdv/2",
+        "data": None,
+        "columns": [
+            {
+                "name": "node_id",
+                "type": "varchar",
+                "typeSignature": {
+                    "typeArguments": [],
+                    "arguments": [{"kind": "LONG_LITERAL", "value": 2147483647}],
+                    "literalArguments": [],
+                    "rawType": "varchar",
+                },
+            },
+            {
+                "name": "http_uri",
+                "type": "varchar",
+                "typeSignature": {
+                    "typeArguments": [],
+                    "arguments": [{"kind": "LONG_LITERAL", "value": 2147483647}],
+                    "literalArguments": [],
+                    "rawType": "varchar",
+                },
+            },
+            {
+                "name": "node_version",
+                "type": "varchar",
+                "typeSignature": {
+                    "typeArguments": [],
+                    "arguments": [{"kind": "LONG_LITERAL", "value": 2147483647}],
+                    "literalArguments": [],
+                    "rawType": "varchar",
+                },
+            },
+            {
+                "name": "coordinator",
+                "type": "boolean",
+                "typeSignature": {
+                    "typeArguments": [],
+                    "arguments": [],
+                    "literalArguments": [],
+                    "rawType": "boolean",
+                },
+            },
+            {
+                "name": "state",
+                "type": "varchar",
+                "typeSignature": {
+                    "typeArguments": [],
+                    "arguments": [{"kind": "LONG_LITERAL", "value": 2147483647}],
+                    "literalArguments": [],
+                    "rawType": "varchar",
+                },
+            },
+        ],
```

Comment:
shorten this to a single column for example, our test doesn't depend on so many being present.